tests/test_slim_engine.py

```diff
@@ -67,14 +67,14 @@ def test_recombination_map(self):
     def test_simulate(self):
         engine = stdpopsim.get_engine("slim")
         species = stdpopsim.get_species("AraTha")
-        # Integral length (10kb) to hit specific code path in slim_engine.py
-        contig = species.get_contig("chr5", length_multiplier=10000/26975502)
+        contig = species.get_contig("chr5", length_multiplier=0.001)
         model = stdpopsim.PiecewiseConstantSize(species.population_size)
         model.generation_time = species.generation_time
         samples = model.get_samples(10)
         ts = engine.simulate(
                 demographic_model=model, contig=contig, samples=samples)
         self.assertEqual(ts.num_samples, 10)
+        self.assertEqual(sum(1 for t in ts.trees() if t.num_roots != 1), 0)
```

Comment:
``self.assertTrue(all(tree.num_roots == 1 for tree in ts.trees())``  possibly clearer?

Reply:
Yep, agreed.